### PR TITLE
Draft: Size for markers

### LINKED ISSUE #4024: Marker items have a setSymbolSize but it is not propagated
If you set `marker.setSymbolSize(20)` it is the same as doing nothing, the marker will stay at size 10.0 (size 11 in the opengl backend)

```
marker.setSymbolSize(20)  # noop
```

### PR DESCRIPTION
**Pull request by vallsv**
Closes #4024

This API was exposed but not propagated the the backend.

Notice that the default marker size was not the same in mpl and opengl.
That's something which have to be checked.

Changelog: 
- Markers now supports `setSymbolSize`

### CODE REVIEW

File: src/silx/gui/plot/backends/BackendOpenGL.py [hunk 1]
```diff
@@ -1135,6 +1138,7 @@ def addMarker(
             text,
             color,
             symbol,
+            int(symbolsize + 0),
```
Comment (t20100) [inline src/silx/gui/plot/backends/BackendOpenGL.py:1141]: Cast to int is not needed, a `float` is used internally:

```suggestion
            symbolsize,
```

**Review by t20100 — APPROVED** [review]

**Comment by vallsv** [comment]
Changed as draft, i dont expect it to be merged.

**Comment by t20100** [comment]
It's not a big PR and it fixes an issue, so it can go in 2.0 for me

**Comment by t20100** [comment]
CI fails, let's keep this for later indeed

**Comment by t20100** [comment]
Closing as rebased and updated in PR  #4181

**Pull request closed without merging**